carAPI/src/main/java/com/everis/boundary/CarResource.java

```diff
@@ -21,26 +22,23 @@
 import javax.ws.rs.core.Response.Status;
 import javax.ws.rs.core.UriInfo;
 
-import org.apache.log4j.Logger;
-
 import com.everis.control.CarService;
 import com.everis.entity.Car;
+import com.everis.utils.LoggerInterceptor;
 
 
 @Path("/cars")
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
+@Interceptors(LoggerInterceptor.class)
```

Comment:
Yeah, this a way to bind an interceptor, but usually we use custom annotations like here:

https://javaee.github.io/tutorial/interceptors002.html